concept/validation/schema-validation.feature

```diff
@@ -11,490 +11,734 @@ Feature: Schema validation
+    When create relation type: rel5
+    When relation(rel5) create role: role00
+    When transaction commits
+
+    When connection open schema transaction for database: typedb
+    Then relation(rel2) get role(role2) get supertype: rel00:role00
+    Then relation(rel2) set supertype: rel5; fails
+
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then relation(rel2) get supertype: rel4
+    Then relation(rel2) get role(role2) get supertype: rel00:role00
+    When relation(rel4) set supertype: rel5
+    Then transaction commits; fails
+
+    When connection open schema transaction for database: typedb
+    Then relation(rel2) get supertype: rel4
+    Then relation(rel2) get role(role2) get supertype: rel00:role00
+    When relation(rel4) set supertype: rel01
+    Then transaction commits; fails
 
 
   Scenario: Deleting a role does not leave dangling 'plays' declarations
     Given create relation type: rel0
-    Given relation(rel0) set relates role: role0
-    Given relation(rel0) set relates role: extra_role
+    Given relation(rel0) create role: role0
+    Given relation(rel0) create role: extra_role
     Given create entity type: ent0
     Given entity(ent0) set plays: rel0:role0
     Given transaction commits
 
-    When session opens transaction of type: write
-    When entity(ent0) get playing roles contain:
+    When connection open schema transaction for database: typedb
+    When entity(ent0) get plays contain:
       | rel0:role0 |
-    When relation(rel0) unset related role: role0
-    Then entity(ent0) get playing roles do not contain:
+    When relation(rel0) delete role: role0
+    Then entity(ent0) get plays do not contain:
       | rel0:role0 |
     Then transaction commits
 
+    When connection open read transaction for database: typedb
+    Then entity(ent0) get plays do not contain:
+      | rel0:role0 |
+
 
   Scenario: Deleting an attribute-type does not leave dangling 'owns' declarations
-    Given create attribute type: attr0, with value type: string
+    Given create attribute type: attr0
+    Given attribute(attr0) set value type: string
     Given create entity type: ent0
-    Given entity(ent0) set owns attribute type: attr0
+    Given entity(ent0) set owns: attr0
     Given transaction commits
 
-    When session opens transaction of type: write
-    When entity(ent0) get owns attribute types contain:
+    When connection open schema transaction for database: typedb
+    When entity(ent0) get owns contain:
       | attr0 |
     When delete attribute type: attr0
-    Then entity(ent0) get owns attribute types do not contain:
+    Then entity(ent0) get owns do not contain:
       | attr0 |
     Then transaction commits
 
+    When connection open read transaction for database: typedb
+    Then entity(ent0) get owns do not contain:
+      | attr0 |
+
 
   Scenario: The schema does not contain redundant owns declarations
```

Comment:
100% have it in owns.feature, but may recheck if it covers all the cases from here